### DISCUSSION TITLE
Unable to read Git repository contents

### DISCUSSION THREAD

**Discussion by sasikumarevature** [community/community#45509] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I am getting this error  **{"message":"Unable to read Git repository contents. We've notified our support staff. If this error persists, or if you have any questions, please contact us.","documentation_url":"https://support.github.com/contact"}**  while creating file in repo through api ..

why this happend, and what is the solution for this issue.

**Comment by airtower-luna** [comment, marked as answer]
The message sounds like it's an internal error, not something you or us other users can solve. If the issue persists, [file a bug report](https://support.github.com/contact/bug-report) to support.

**Reply by sasikumarevature** [reply]
Any other solution is there ?

**Reply by jge162** [reply]
could be many things, repository's underlying storage, repository's permissions or configuration, security issue with the account or organization. luna has given a great answer. consider filling a bug report.

**Reply by sasikumarevature** [reply]
Thank you guys
